sktime/utils/_testing/estimator_checks.py

```diff
@@ -216,6 +217,11 @@ def get_tag(est, tag_name, tag_value_default=None):
             return True
         return get_tag(est, "capability:pred_int", False)
     # skip transform for forecasters that have it - pipelines
-    if method == "transform" and isinstance(est, BaseForecaster):
+    if method == "transform" and scitype(est) in (
```

Comment:
this changes the logic, it is not just a refactor - I assume you have good reason?

Reply:
When testing the graphical pipeline there would be the case that a classification pipeline is constructed with a `transform` method and a transformation pipeline with a `predict` call. Perhaps, it would be better to move this into a separate PR.

Reply:
I think it's fine, I would say, just update the comment with examples. The example for forecasting are reduction forecasters, they may have transform.